diff --git a/node_modules/@ledgerhq/react-native-hid/android/src/main/java/com/ledgerwallet/hid/ReactHIDModule.java b/node_modules/@ledgerhq/react-native-hid/android/src/main/java/com/ledgerwallet/hid/ReactHIDModule.java
index 349bf39..92b5b1e 100644
--- a/node_modules/@ledgerhq/react-native-hid/android/src/main/java/com/ledgerwallet/hid/ReactHIDModule.java
+++ b/node_modules/@ledgerhq/react-native-hid/android/src/main/java/com/ledgerwallet/hid/ReactHIDModule.java
@@ -7,6 +7,7 @@ import android.content.Intent;
 import android.content.IntentFilter;
 import android.hardware.usb.UsbDevice;
 import android.hardware.usb.UsbManager;
+import android.os.Build;
 
 import androidx.annotation.NonNull;
 
@@ -58,6 +59,11 @@ public class ReactHIDModule extends ReactContextBaseJavaModule {
                         .emit(event, buildMapFromDevice(device));
             }
         };
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            getReactApplicationContext().registerReceiver(receiver, filter, Context.RECEIVER_NOT_EXPORTED);
+            return;
+        }
+
         getReactApplicationContext().registerReceiver(receiver, filter);
     }
 
@@ -213,6 +219,11 @@ public class ReactHIDModule extends ReactContextBaseJavaModule {
                 unregisterReceiver(this);
             }
         };
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            getReactApplicationContext().registerReceiver(receiver, intFilter, Context.RECEIVER_NOT_EXPORTED);
+            return;
+        }
+
         getReactApplicationContext().registerReceiver(receiver, intFilter);
     }
 
